themes/beaver/layouts/partials/blog/related-posts.html

```diff
@@ -1,10 +1,12 @@
 {{ $blogPosts := where .Site.RegularPages "Section" "blog" }}
 {{ $related := $blogPosts.Related . | first 5 }}
 {{ with $related }}
+<aside style="margin-top:4rem">
 <h3>Read next</h3>
 <ul>
  {{ range . }}
  <li><a href="{{ .RelPermalink }}">{{ .LinkTitle }}</a></li>
  {{ end }}
 </ul>
-{{ end }}
\ No newline at end of file
+{{ end }}
+</aside>
```

Comment:
please add indentions for html tags to make it easier to read

Reply:
++